### ISSUE TITLE
Android guidelines - Non-applicable points

### ISSUE THREAD

**Issue by Aksi0m** [infinum/accessibility-mobile-standards#45]
Review the given WCAG 2.2 Level A/AA points and refine if needed:

- [ ] [1.4.13](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=1413)
- [ ] [2.1.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=211)
- [ ] [2.1.2](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=212)
- [ ] [2.1.4](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=214)
- [ ] [2.5.1](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=251)
- [ ] [2.5.2](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=252)
- [ ] [3.2.6](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa&showtechniques=326)

Your main reference points should be:
1. The [WCAG website](https://www.w3.org/WAI/WCAG22/quickref/?currentsidebar=%23col_customize&levels=aaa) 
2. The iOS implementation of the same point in this repository

**If you find some useful articles, videos, or any resource please share it with the rest of us in our dedicated slack channel.
For any questions you can ask Renato, Petra and Majcen**
